src/app/core/admin/admin-dashboard/admin-dashboard.component.html

```diff
@@ -0,0 +1,109 @@
+  <app-view-title
+    class="admin-ui-title"
+    i18n="page title"
+    [displayInPlace]="true"
+  >
+    Edit Dashboard Widgets
+  </app-view-title>
+
+  <div class="flex-row gap-small save-buttons">
+    <button mat-raised-button color="accent" (click)="save()" i18n>Save</button>
+    <button mat-stroked-button (click)="cancel()" i18n>Cancel</button>
+  </div>
+</div>
+
+<div class="hint-banner" i18n>
+  You can customize the widgets on your dashboard to highlight the information
+  that is most useful to you. You can personalize the dashboard to show the key
+  insights you need.
+</div>
+
+<div class="overall-container flex-row" [class.disabled]="isDisabled">
+  <!-- WIDGET PREVIEW AREA -->
+  <div
+    cdkDropList
+    [cdkDropListData]="dashboardConfig.widgets"
+    (cdkDropListDropped)="drop($event)"
+    cdkDropListOrientation="mixed"
+    class="widgets-container drop-list"
+  >
+    @for (
+      widgetConfig of dashboardConfig.widgets;
+      track widgetConfig;
+      let i = $index
+    ) {
+      <div class="widget-container">
+        <mat-card
+          cdkDrag
+          cdkDragBoundary=".overall-container"
+          class="widget-preview-card"
+        >
+          <fa-icon
+            icon="grip-vertical"
+            size="xl"
+            class="drag-handle"
+            cdkDragHandle
+            matTooltip="You can rearrange the dashboard sections to highlight the most important information first."
+            i18n-matTooltip
+          ></fa-icon>
+
+          <fa-icon
+            icon="times"
+            class="widget-remove-icon"
+            (click)="removeWidget(i)"
+            matTooltip="Remove this widget"
+            i18n-matTooltip
+          ></fa-icon>
+
+          <mat-card-content class="widget-preview-section">
+            <!-- Widget Content with Overlay Edit Button -->
+            <div class="widget-content-wrapper">
+              <ng-template [appDynamicComponent]="widgetConfig"></ng-template>
+
+              <!-- Centered Edit Button Overlay -->
+              <div class="field-hover-buttons">
+                <button
+                  mat-stroked-button
+                  color="accent"
+                  class="field-edit-button"
+                  (click)="editWidget(widgetConfig, i)"
+                  i18n="Button label"
+                >
+                  Edit Widget
+                </button>
+              </div>
+            </div>
+          </mat-card-content>
+        </mat-card>
+      </div>
+    }
+
+    <!-- DROP AREA: ADD NEW WIDGET -->
+    <mat-card class="widget-preview-card drop-area-hint">
+      <mat-card-content class="add-widget-content">
+        <span i18n="Header of drag and drop box">
+          click here to add new widget
+        </span>
+        <fa-icon
+          icon="question-circle"
+          matTooltip="Add widgets to customize your dashboard. You can drag and drop to reorder them."
+          i18n-matTooltip
+        ></fa-icon>
```

Comment:
```suggestion
```

I have added a very similar explanation to the top (where it'll be more easily visible). So I'd remove that here